diff --git a/node_modules/gatsby/dist/schema/graphql-engine/bundle-webpack.js b/node_modules/gatsby/dist/schema/graphql-engine/bundle-webpack.js
index 71dca09..8f96f82 100644
--- a/node_modules/gatsby/dist/schema/graphql-engine/bundle-webpack.js
+++ b/node_modules/gatsby/dist/schema/graphql-engine/bundle-webpack.js
@@ -87,7 +87,9 @@ async function createGraphqlEngineBundle(rootDir, reporter, isVerbose) {
       }
 
       return acc;
-    }, {})],
+    }, {
+      "fs-extra": `global _actualFsExtraWrapper`,
+    })],
     module: {
       rules: [{
         oneOf: [{
@@ -152,6 +154,7 @@ async function createGraphqlEngineBundle(rootDir, reporter, isVerbose) {
       extensions,
       alias: {
         ".cache": process.cwd() + `/.cache/`,
+        "actual-fs-extra": `fs-extra`,
         [require.resolve(`gatsby-cli/lib/reporter/loggers/ink/index.js`)]: false,
         inquirer: false,
         // only load one version of lmdb
@@ -166,7 +169,8 @@ async function createGraphqlEngineBundle(rootDir, reporter, isVerbose) {
       "process.env.GATSBY_SKIP_WRITING_SCHEMA_TO_FILE": `true`,
       "process.env.NODE_ENV": JSON.stringify(`production`),
       SCHEMA_SNAPSHOT: JSON.stringify(schemaSnapshotString),
-      "process.env.GATSBY_LOGGER": JSON.stringify(`yurnalist`)
+      "process.env.GATSBY_LOGGER": JSON.stringify(`yurnalist`),
+      "process.env.GATSBY_SLICES": JSON.stringify(!!process.env.GATSBY_SLICES)
     }), ((_process$env$GATSBY_W = process.env.GATSBY_WEBPACK_LOGGING) === null || _process$env$GATSBY_W === void 0 ? void 0 : _process$env$GATSBY_W.includes(`query-engine`)) && new _webpackLogging.WebpackLoggingPlugin(rootDir, reporter, isVerbose)].filter(Boolean)
   });
   return new Promise((resolve, reject) => {
diff --git a/node_modules/gatsby/dist/utils/engines-fs-provider.js b/node_modules/gatsby/dist/utils/engines-fs-provider.js
index 3cb7a2a..e3c9e15 100644
--- a/node_modules/gatsby/dist/utils/engines-fs-provider.js
+++ b/node_modules/gatsby/dist/utils/engines-fs-provider.js
@@ -11,11 +11,17 @@ exports.__esModule = true;
 // eslint-disable-next-line @typescript-eslint/naming-convention
 if (global._fsWrapper) {
   global._actualFsWrapper = global._fsWrapper;
+  global._actualFsExtraWrapper = {
+    ...require(`actual-fs-extra`),
+    copySync: global._fsWrapper.copySync,
+  }
 } else {
   // fs alternative not provided - falling back to regular fs
   global._actualFsWrapper = __non_webpack_require__(`fs`);
+  global._actualFsExtraWrapper = require(`actual-fs-extra`)
 } // hydrate webpack module cache (consume global, so it's not lazy)
 
 
 require(`fs`); // https://stackoverflow.com/a/59499895
+require(`fs-extra`)
 //# sourceMappingURL=engines-fs-provider.js.map
\ No newline at end of file
diff --git a/node_modules/gatsby/dist/utils/page-ssr-module/bundle-webpack.js b/node_modules/gatsby/dist/utils/page-ssr-module/bundle-webpack.js
index 9ea1a6b..a3c613d 100644
--- a/node_modules/gatsby/dist/utils/page-ssr-module/bundle-webpack.js
+++ b/node_modules/gatsby/dist/utils/page-ssr-module/bundle-webpack.js
@@ -127,6 +127,7 @@ async function createPageSSRBundle({
     resolve: {
       extensions,
       alias: {
+        "actual-fs-extra": `fs-extra`,
         ".cache": `${rootDir}/.cache/`,
         [require.resolve(`gatsby-cli/lib/reporter/loggers/ink/index.js`)]: false,
         inquirer: false
