Index: node_modules/gatsby/dist/schema/graphql-engine/bundle-webpack.js
===================================================================
--- node_modules/gatsby/dist/schema/graphql-engine/bundle-webpack.js
+++ node_modules/gatsby/dist/schema/graphql-engine/bundle-webpack.js
@@ -77,8 +77,9 @@
     externals: [`cbor-x`, // optional dep of lmdb-store, but we are using `msgpack` (default) encoding, so we don't need it
     `babel-runtime/helpers/asyncToGenerator`, // undeclared dep of yurnalist (but used in code path we don't use)
     `electron`, // :shrug: `got` seems to have electron specific code path
     _module.default.builtinModules.reduce((acc, builtinModule) => {
+      console.log(`OUR LOG`, { builtinModule })
       if (builtinModule === `fs`) {
         acc[builtinModule] = `global _actualFsWrapper`;
         acc[`node:${builtinModule}`] = `global _actualFsWrapper`;
       } else {
